Source/Charts/Data/Implementations/Standard/ChartDataEntryBase.swift

```diff
@@ -106,27 +87,16 @@ open class ChartDataEntryBase: NSObject
     }
 }
 
-public func ==(lhs: ChartDataEntryBase, rhs: ChartDataEntryBase) -> Bool
-{
-    if lhs === rhs
+// MARK: Equatable
+extension ChartDataEntryBase/*: Equatable*/ {
```

Comment:
Why is equatable commented out?

Reply:
Because `NSObject` adopts `Equatable` and you can't declare conformance twice. When Obj-C support is dropped, then we will need to uncomment it.